cdap-api-common/src/main/java/co/cask/cdap/api/common/Bytes.java

```diff
@@ -257,6 +257,20 @@ public static String toHexString(byte [] bytes) {
   }
 
   /**
+   * Returns a byte[] by decoding from the given hexadecimal string. The string must be have an even length.
+   */
+  public static byte[] fromHexString(String str) {
```

Comment:
I remember that test too, it's probably hidden inside some table scanner test. Let me dig that out